lib/wrap.js

```diff
@@ -56,7 +56,13 @@ function metrics(main) {
         }
 
         try {
-            return await main(params);
+            // send activation_start metrics and execute main action in parallel
+            const activationStartedMetricsPromise = metrics.activationStarted();
+            const mainPromise = main(params);
+            await Promise.all([activationStartedMetricsPromise, mainPromise]);
```

Comment:
`Promise.all` or `Promise.allSettled`? What happens if the promise sending the NR event is rejected?